docs/examples/cookbook/drag-and-drop.md

```diff
@@ -607,7 +607,7 @@ Each entry dynamically picks one of the two destinations based on its
 completion status.
 
 ```Lua
-			Parent = scope:Computed(function(use)
+			Parent = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
			Parent = scope:Computed(function(use)
```

Reply:
reverted